src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -16,63 +16,49 @@ import collaboration_discussion_board_new from 'images/collaboration-discussion-
+Once a question is answered or an issue is resolved, you can mark a discussion as "Closed" via the **More Actions** menu on the discussion thread. Your team can still add comments to the discussion, just like in a Jira ticket, and it will not automatically reopen a discussion. If you want to reopen a discussion, you can do it manually, like you closed it.
 
 <img
-  title="Close and reopen discussions "
-  alt="A screenshot of the UI that depicts closing discussions."
+  title= "Close and reopen discussions"
+  alt= "A screenshot of the UI that depicts closing discussions."
   src={close_and_reopen_discussions}
-  style={{ maxWidth: '60%' }}
+  style={{ maxWidth: '60%'}}
 />
 
-## Follow and unfollow discussions [#follow-unfollow]
-
-When you add a comment to a discussion or if you're at-mentioned, you'll automatically be assigned to follow the discussion. You'll know you've been assinged to a discussion because you will see the bell icon on the discussion and on the discussions board. Following a discussion means you'll get email notifications and the Discussions board UMI (see next section) will increment when new comments are added to a discussion. If you want to unfollow a discussion that's also available via the more actions menu on the discussion (see screenshot in doc above).  If there is a discussion you are interested in that you're not following but would like to, you can manually follow the same way. 
+## Following discussions  [#follow-unfollow]
 
-## The discussions board [#discussions-board]
-
-<img
-  title="Discussions board"
-  alt="A screenshot of the UI that depicts the discussions board."
-  src={collaboration_discussion_board_new}
-/>
+Whenever you comment or get mentioned, you'll automatically follow the discussion. This means you'll receive email notifications when new comments are added.
 
-Go to the discussions board to see a complete history of the discussion threads in your organization. On the left nav, the UMI on the Discussions link indicates discussion threads you are following that have unread comments. On the Discussions page, click a discussion thread to jump to the UI where that conversation is happening.
+Stay in the loop with discussions:
 
-We know that your team uses a wide variety of tools to communicate, but when discussions are spread across platforms and teams, it's easy to lose institutional knowledge. You can now use the discussion board to find active discussions as well as search the archive of shared knowledge. Our discussion board helps you find answers to questions and research how issues and problems were solved in the past.
+* Look for the bell icon on the discussion to see if you're following.
+* Want to stop receiving notifications? Unfollow the discussion from the "More Actions" menu.
+* Not interested in a specific discussion? Don't worry; you can easily follow it manually through the same menu.
```

Comment:
@homelessbirds I'm not really sure what it means to following a discussion manually. What is this bulleted point trying to say?

Reply:
Good callout! I fixed this.